### PR TITLE
feat(ui): Introduce NavbarAccordion

### PR DESCRIPTION
**Pull request by devdesrel**

### CODE REVIEW

File: packages/lab/src/navbar/NavbarAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  label,
+  icon,
+  gutter,
+  items,
+  onSelect,
+}: NavbarAccordionProps): ReactElement {
+  const uid = useUID();
+  const { isExpanded: isMenuExpanded } = useNavbarContext();
+
+  const [expanded, setExpanded] = useState(false);
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarAccordion.tsx:96]: Lets use same naming `isExpanded`

File: packages/lab/src/navbar/NavbarItem.tsx [hunk 2]
```diff
@@ -58,6 +58,10 @@ const NavbarItemRoot = styled.div`
     color: ${Color.White};
     border-left-color: ${Color.Blue300};
   }
+
+  &[data-active='true'] {
+    border-left-color: ${Color.Blue300};
+  }
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarItem.tsx:64]: Shouldn't we have same styles from hover and aria-current?

**Review by doniyor2109 — APPROVED** [review]

**Comment by doniyor2109** [comment]
@devdesrel Before merging, I recommend testing it on STMS first. You can do by using Codesandbox version of this changes https://github.com/superdispatch/web-ui/pull/353#issuecomment-1088522778

File: packages/lab/src/navbar/NavbarAccordion.tsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  return (
+    <NavbarAccordionRoot
+      aria-labelledby={uid}
+      gutter={!!gutter}
+      expanded={isExpanded}
+      onClick={(event) => {
+        if (isMenuExpanded || isDrawerOpen) {
+          setExpanded(!isExpanded);
+        } else if (items.length > 0 && !items.some((item) => item.active)) {
+          items[0]?.onClick?.(event);
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarAccordion.tsx:122]: I think we have misscomunicated again)
Can we call `onExpand` here instead?

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: packages/lab/src/navbar/NavbarAccordion.tsx (hunk 3 above)
Comment (devdesrel) [reply packages/lab/src/navbar/NavbarAccordion.tsx:122]: @doniyor2109 should NavbarAccordion receive onExpand as prop from NavbarList?

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**